pyproject.toml

```diff
@@ -11,35 +11,36 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-click = ">=8.1.6"
-curies = ">=0.7.3"
-linkml-runtime = ">=1.5.5"
-pandas = ">1.0.3"
-pansql = { version = ">=0.0.1", extras = ["pansql"] }
-sssom-schema = ">=0.14.0"
-networkx = { version = ">=3.1", extras = ["networkx"] }
-sparqlwrapper = ">=2.0.0"
-validators = ">=0.20.0"
+python = "^3.9"
+click = ">=8.1.7"
+curies = ">=0.7.8"
+pandas = ">=2.2.1"
```

Comment:
Will this cause a problem like it did with https://github.com/mapping-commons/sssom-py/issues/432

cc @kevinschaper

Reply:
It might, but we can solve it on the cat-merge / kgx side this time, I say go for it